.docker/parameters.yml

```diff
@@ -36,7 +36,7 @@ parameters:
             id: aging-special-issue
             text: '<a href="https://elifesciences.org/inside-elife/4f706531/special-issue-call-for-papers-in-aging-geroscience-and-longevity">Read the call for papers</a> for the eLife Special Issue on Aging, Geroscience and Longevity.'
             article_ids:
-              - '26231'
+              - '00569'
```

Comment:
[not an article in `api-dummy](https://github.com/elifesciences/api-dummy/search?q=00569&unscoped_q=00569) so does this work when loading http://localhost:8080/articles/00569? :thinking:

Reply:
Rolled back in 9a46784.

Reply:
The unit test also needs to match, as it stubs the article data but relies on that configuration to decide whether to show the info-bar.